app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_part = command.split()
+
+    if command_part[0] == "cp":
+        with open(
+                command_part[1], "r"
+        ) as file, open(
+            command_part[2], "a"
+        ) as file_copy:
```

Comment:
```suggestion
        with (
            open(command_parts[1], "r") as file, 
            open( command_parts[2], "a") as file_copy
        ):
```

Reply:
when I try to use parentheses I get a syntax error

Reply:
![изображение](https://github.com/mate-academy/py-copy-file/assets/116379690/9aae5498-5db2-46fb-ac22-67064464ce29)

Reply:
@neagarr 
![image](https://github.com/mate-academy/py-copy-file/assets/93856423/a296c7ed-dbb8-4945-9e0f-f0b1fa637cbb)

Reply:
@neagarr maybe you get syntax error because you use old python
I approved this task, but for the future try to use newer versions of python
![image](https://github.com/mate-academy/py-copy-file/assets/93856423/ae419248-85ca-4050-917a-94dbea280f6d)